quic/api/Observer.h

```diff
@@ -29,10 +29,13 @@ class InstrumentationObserver {
         const quic::OutstandingPacket& pkt)
         : lostByTimeout(lostbytimeout),
           lostByReorderThreshold(lostbyreorder),
-          packet(pkt) {}
+          metadata(pkt.metadata),
```

Comment:
For the loss events, we actually want to get the original packet (at least for now). Can you change back to expose the packet?